### PR TITLE
Adjustments to the file CONTRIBUTING.md towards "libui-ng" and a basic .editorconfig

### PR DESCRIPTION
**Pull request by philiplb**
While digging around a bit of how to contribute to libui-ng, I replaced the "old" name "libui" with "libui-ng". And I added a basic .editorconfig for C/C++-projects for some code style help of the editor/IDE.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+charset = utf-8
+trim_trailing_whitespace = true
+end_of_line = lf
+insert_final_newline = true
+
+[Makefile]
+indent_style = tab
+
+[*.{c,h,cpp,hpp}]
+indent_size = 2
```
Comment (cody271) [inline .editorconfig:13]: Needs `indent_style = tab` here as well no?

Comment (philiplb) [reply .editorconfig]: Simplified it a bit and moved the tab style to the C/Cpp files

**Comment by cody271** [comment]
@philiplb Also it appears that you've made this PR off of your fork repo's `master` branch.

While _technically_ this is allowed by GitHub's git workflow, it is strongly discouraged.

`philiplb:master` should be a feature/topic branch, like `philiplb:editorconfig_contributing_md`.

**Comment by philiplb** [comment]
> @philiplb Also it appears that you've made this PR off of your fork repo's `master` branch.
> 
> While _technically_ this is allowed by GitHub's git workflow, it is strongly discouraged.
> 
> `philiplb:master` should be a feature/topic branch, like `philiplb:editorconfig_contributing_md`.

Done in a new PR

**Pull request closed without merging**